### PR TITLE
[MBL-1667] Pledge Redemption String Translations

### PR DESCRIPTION
**Pull request by scottkicks**
# 📲 What

Pulls in the latest string translations and replaces our hard-coded strings.

# 🤔 Why

KS supports multiple countries.

# 🛠 How

run `make strings`
update hardcoded strings

### CODE REVIEW

File: Library/SharedFunctions.swift [hunk 1]
```diff
@@ -661,7 +661,11 @@ public func isRewardDigital(_ reward: Reward?) -> Bool {
     .isAny(of: Reward.Shipping.Preference.none)
 }
 
-public func estimatedShippingText(for reward: Reward, selectedShippingRule: ShippingRule) -> String {
+public func estimatedShippingText(
+  for reward: Reward,
+  project: Project,
+  selectedShippingRule: ShippingRule
+) -> String {
```
Comment (scottkicks) [inline Library/SharedFunctions.swift:668]: I needed to update this method to accept the current Project. This allowed me to use the `Format.currency()` helper which gives us the correct currency symbol and formatting that we need.

File: Library/ViewModels/NoShippingPledgeViewModel.swift [hunk 2]
```diff
@@ -318,8 +318,8 @@ public class NoShippingPledgeViewModel: NoShippingPledgeViewModelType, NoShippin
         guard let rule = shippingRule else { return ("", "") }
 
         return (
-          estimatedShippingText(for: reward, selectedShippingRule: rule),
-          estimatedShippingConversionText(project, reward, selectedShippingRule: rule)
+          estimatedShippingText(for: reward, project: project, selectedShippingRule: rule),
+          estimatedShippingConversionText(for: reward, project: project, selectedShippingRule: rule)
```
Comment (scottkicks) [inline Library/ViewModels/NoShippingPledgeViewModel.swift:322]: related to my [comment](https://github.com/kickstarter/ios-oss/pull/2137/files#r1736468366) above

File: Library/ViewModels/PostCampaignCheckoutViewModel.swift [hunk 3]
```diff
@@ -145,8 +145,8 @@ public class PostCampaignCheckoutViewModel: PostCampaignCheckoutViewModelType,
         guard let rule = shippingRule, let reward = baseReward else { return ("", "") }
 
         return (
-          estimatedShippingText(for: reward, selectedShippingRule: rule),
-          estimatedShippingConversionText(project, reward, selectedShippingRule: rule)
+          estimatedShippingText(for: reward, project: project, selectedShippingRule: rule),
+          estimatedShippingConversionText(for: reward, project: project, selectedShippingRule: rule)
```
Comment (scottkicks) [inline Library/ViewModels/PostCampaignCheckoutViewModel.swift:149]: also related to my [comment](https://github.com/kickstarter/ios-oss/pull/2137/files#r1736468366) above

**Review by ifosli — APPROVED** [review]
Code changes look great! I re-recorded the failing snapshots but lmk if you want me to re-record all of them!

**Pull request merged**